src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -38,8 +38,8 @@
 
         // Index Motor Speeds
 
-        public static double INDEX_IN_SPEED = 0.2; // will change values later
-        public static double INDEX_OUT_SPEED = -0.3; // will also change later
+        public static double INDEX_IN_SPEED = 0.5; // will change values later
+        public static double INDEX_OUT_SPEED = -0.5; // will also change later
```

Comment:
Where did these values come from?

Reply:
testing

Reply:
To make sure there is no conflict, let's remove these from the changes.  
Index / intake will be tuned separately.